django/db/backends/oracle/base.py

```diff
@@ -399,12 +403,11 @@ class FormatStylePlaceholderCursor(object):
     """
     charset = 'utf-8'
 
-    def __init__(self, connection):
+    def __init__(self, connection, chunk_size=constants.GET_ITERATOR_CHUNK_SIZE):
         self.cursor = connection.cursor()
         # Necessary to retrieve decimal values without rounding error.
         self.cursor.numbersAsStrings = True
-        # Default arraysize of 1 is highly sub-optimal.
```

Comment:
I removed this comment because it is no longer valid, `cx_Oracle`'s default `arraysize` [is now 50](http://cx-oracle.readthedocs.io/en/latest/cursor.html#Cursor.arraysize).